<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>F_Contenant</class>
 <widget class="QDialog" name="F_Contenant">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>171</width>
     <height>101</height>
    </rect>
   </property>
   <property name="text">
    <string>Veuillez entrer les donnees 
concernant les contenants propre 
au lot précédemment entré.
Renseigner la masse nette, 
le numéro du contenant et 
précisez si il s'agit d'un contenant 
fractionné ou complet.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btAjoutContenant">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter contenant</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btSortirForm">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>250</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Sortir du formulaire</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdContComplet">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Complet</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rdContFractionne">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>210</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fractionné</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Libelle Article:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Numéro de Lot:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Masse totale du lot:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbLibArt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>30</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbNumLotArt">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbMasseTotaleLot">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>--</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFormContenant1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFormContenant2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbFormContenant3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>241</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>357</width>
     <height>81</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Numéro contenant:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leNumCont"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Masse nette contenant:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="leMasseNetteCont"/>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>kg</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
